tests/unit/server/bundle_store_test.py

```diff
@@ -0,0 +1,98 @@
+from codalab.common import StorageType, StorageFormat
+from tests.unit.server.bundle_manager import BaseBundleManagerTest
+
+
+class BundleStoreTest(BaseBundleManagerTest):
+    def test_add_bundle_location(self):
+        """Create a bundle location for a bundle store."""
```

Comment:
can you update this test description so it's reflective of what's going on in the test?